src/components/vuestic-components/va-option-list/VaOptionList.demo.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <VbDemo>
+    <VbCard
+      title="Options as array of string"
+    >
+      <va-option-list
+        :options="baseData.options"
+        :default-value="baseData.options[0]"
+        color="primary"
+        type="radio"
+        v-model="baseData.selected"
+      />
+      Selected {{ baseData.selected }}
+    </VbCard>
+    <VbCard
+      title="Disable all"
+    >
+      <va-option-list
+        :options="baseData.options"
+        type="radio"
+        disabled
+      />
+    </VbCard>
+    <VbCard
+      title="Left label"
+    >
+      <va-option-list
+        :options="baseData.options"
+        type="radio"
+        color="primary"
+        left-label
+      />
+    </VbCard>
+    <VbCard
+      title="Readonly"
+    >
+      <va-option-list
+        :options="baseData.options"
+        :default-value="baseData.options[0]"
+        color="danger"
+        type="radio"
+        readonly
+      />
+    </VbCard>
+    <VbCard
+      title="Use more complex data"
+    >
+      <va-option-list
+        :options="complexData.options"
+        v-model="complexData.selectedValue"
+        type="radio"
+        color="warning"
+      />
+      Selected {{ complexData.selectedValue }}
+    </VbCard>
+
+    <VbCard
+      title="Return whole object"
+    >
+      <va-option-list
+        :options="complexData.options"
+        v-model="complexData.selectedObject"
+        :default-value="complexData.options[1]"
+        type="radio"
+        output-object
+        color="warning"
+      />
+      Selected {{ complexData.selectedObject }}
+    </VbCard>
+
+    <VbCard
+      title="Using slot"
+    >
+      <va-option-list
+        :options="complexData.options"
+        v-model="complexData.selectedValueFromSlot"
+        :default-value="complexData.options[0]"
+        color="danger"
+      >
+        <template v-slot="{ props }">
+          <va-radio
+            :option="props.option"
```

Comment:
Maybe just show all props as object? `<pre>{{ props }}</pre>`. Up to you though.